[2024-09-09T10:17:57.747+0000] {processor.py:186} INFO - Started process (PID=53) to work on /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:17:57.747+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/weatherbit_dag.py for tasks to queue
[2024-09-09T10:17:57.749+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:17:57.749+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:17:57.782+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:17:57.777+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/weatherbit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/weatherbit_dag.py", line 3, in <module>
    from operators.weatherbit_operator import WeatherbitOperator
  File "/opt/airflow/plugins/operators/weatherbit_operator.py", line 2, in <module>
    from hooks.weatherbit_hook import WeatherbitHook
ModuleNotFoundError: No module named 'hooks'
[2024-09-09T10:17:57.784+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:17:57.800+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/weatherbit_dag.py took 0.060 seconds
[2024-09-09T10:18:27.907+0000] {processor.py:186} INFO - Started process (PID=60) to work on /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:18:27.907+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/weatherbit_dag.py for tasks to queue
[2024-09-09T10:18:27.909+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:18:27.909+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:18:27.932+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:18:27.928+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/weatherbit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/weatherbit_dag.py", line 3, in <module>
    from operators.weatherbit_operator import WeatherbitOperator
  File "/opt/airflow/plugins/operators/weatherbit_operator.py", line 2, in <module>
    from hooks.weatherbit_hook import WeatherbitHook
ModuleNotFoundError: No module named 'hooks'
[2024-09-09T10:18:27.933+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:18:27.948+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/weatherbit_dag.py took 0.045 seconds
[2024-09-09T10:18:58.079+0000] {processor.py:186} INFO - Started process (PID=67) to work on /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:18:58.080+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/weatherbit_dag.py for tasks to queue
[2024-09-09T10:18:58.081+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:18:58.081+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:18:59.131+0000] {processor.py:925} INFO - DAG(s) 'weatherbit_dag' retrieved from /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:18:59.289+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:18:59.288+0000] {override.py:1858} INFO - Created Permission View: can delete on DAG:weatherbit_dag
[2024-09-09T10:18:59.297+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:18:59.297+0000] {override.py:1858} INFO - Created Permission View: can read on DAG:weatherbit_dag
[2024-09-09T10:18:59.303+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:18:59.303+0000] {override.py:1858} INFO - Created Permission View: can edit on DAG:weatherbit_dag
[2024-09-09T10:18:59.303+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:18:59.303+0000] {dag.py:3211} INFO - Sync 1 DAGs
[2024-09-09T10:18:59.313+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:18:59.313+0000] {dag.py:3234} INFO - Creating ORM DAG for weatherbit_dag
[2024-09-09T10:18:59.320+0000] {processor.py:211} ERROR - Got an exception! Propagating...
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/dag_processing/processor.py", line 207, in _run_file_processor
    _handle_dag_file_processing()
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/dag_processing/processor.py", line 188, in _handle_dag_file_processing
    result: tuple[int, int, int] = dag_file_processor.process_file(
                                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/session.py", line 97, in wrapper
    return func(*args, session=session, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/dag_processing/processor.py", line 942, in process_file
    serialize_errors = DagFileProcessor.save_dag_to_db(
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/api_internal/internal_api_call.py", line 139, in wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/session.py", line 97, in wrapper
    return func(*args, session=session, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/dag_processing/processor.py", line 982, in save_dag_to_db
    import_errors = DagBag._sync_to_db(dags=dags, processor_subdir=dag_directory, session=session)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/session.py", line 94, in wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 707, in _sync_to_db
    for attempt in run_with_db_retries(logger=log):
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/tenacity/__init__.py", line 443, in __iter__
    do = self.iter(retry_state=retry_state)
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/tenacity/__init__.py", line 376, in iter
    result = action(retry_state)
             ^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/tenacity/__init__.py", line 398, in <lambda>
    self._add_action_func(lambda rs: rs.outcome.result())
                                     ^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/concurrent/futures/_base.py", line 449, in result
    return self.__get_result()
           ^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/concurrent/futures/_base.py", line 401, in __get_result
    raise self._exception
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 723, in _sync_to_db
    DAG.bulk_write_to_db(dags.values(), processor_subdir=processor_subdir, session=session)
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/session.py", line 94, in wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dag.py", line 3290, in bulk_write_to_db
    orm_dag.calculate_dagrun_date_fields(dag, last_automated_data_interval)
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dag.py", line 4130, in calculate_dagrun_date_fields
    next_dagrun_info = dag.next_dagrun_info(last_automated_data_interval)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dag.py", line 1155, in next_dagrun_info
    restriction = self._time_restriction
                  ^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/functools.py", line 993, in __get__
    val = self.func(instance)
          ^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dag.py", line 1194, in _time_restriction
    earliest = timezone.coerce_datetime(min(start_dates))
                                        ^^^^^^^^^^^^^^^^
TypeError: '<' not supported between instances of 'str' and 'DateTime'
[2024-09-09T10:19:20.047+0000] {processor.py:186} INFO - Started process (PID=47) to work on /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:19:20.047+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/weatherbit_dag.py for tasks to queue
[2024-09-09T10:19:20.048+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:19:20.048+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:19:20.071+0000] {processor.py:925} INFO - DAG(s) 'weatherbit_dag' retrieved from /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:19:20.079+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:19:20.079+0000] {dag.py:3211} INFO - Sync 1 DAGs
[2024-09-09T10:19:20.089+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:19:20.089+0000] {dag.py:3234} INFO - Creating ORM DAG for weatherbit_dag
[2024-09-09T10:19:20.096+0000] {processor.py:211} ERROR - Got an exception! Propagating...
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/dag_processing/processor.py", line 207, in _run_file_processor
    _handle_dag_file_processing()
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/dag_processing/processor.py", line 188, in _handle_dag_file_processing
    result: tuple[int, int, int] = dag_file_processor.process_file(
                                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/session.py", line 97, in wrapper
    return func(*args, session=session, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/dag_processing/processor.py", line 942, in process_file
    serialize_errors = DagFileProcessor.save_dag_to_db(
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/api_internal/internal_api_call.py", line 139, in wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/session.py", line 97, in wrapper
    return func(*args, session=session, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/dag_processing/processor.py", line 982, in save_dag_to_db
    import_errors = DagBag._sync_to_db(dags=dags, processor_subdir=dag_directory, session=session)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/session.py", line 94, in wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 707, in _sync_to_db
    for attempt in run_with_db_retries(logger=log):
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/tenacity/__init__.py", line 443, in __iter__
    do = self.iter(retry_state=retry_state)
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/tenacity/__init__.py", line 376, in iter
    result = action(retry_state)
             ^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/tenacity/__init__.py", line 398, in <lambda>
    self._add_action_func(lambda rs: rs.outcome.result())
                                     ^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/concurrent/futures/_base.py", line 449, in result
    return self.__get_result()
           ^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/concurrent/futures/_base.py", line 401, in __get_result
    raise self._exception
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 723, in _sync_to_db
    DAG.bulk_write_to_db(dags.values(), processor_subdir=processor_subdir, session=session)
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/session.py", line 94, in wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dag.py", line 3290, in bulk_write_to_db
    orm_dag.calculate_dagrun_date_fields(dag, last_automated_data_interval)
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dag.py", line 4130, in calculate_dagrun_date_fields
    next_dagrun_info = dag.next_dagrun_info(last_automated_data_interval)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dag.py", line 1155, in next_dagrun_info
    restriction = self._time_restriction
                  ^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/functools.py", line 993, in __get__
    val = self.func(instance)
          ^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dag.py", line 1194, in _time_restriction
    earliest = timezone.coerce_datetime(min(start_dates))
                                        ^^^^^^^^^^^^^^^^
TypeError: '<' not supported between instances of 'str' and 'DateTime'
[2024-09-09T10:19:50.200+0000] {processor.py:186} INFO - Started process (PID=54) to work on /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:19:50.200+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/weatherbit_dag.py for tasks to queue
[2024-09-09T10:19:50.202+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:19:50.201+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:19:50.226+0000] {processor.py:925} INFO - DAG(s) 'weatherbit_dag' retrieved from /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:19:50.243+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:19:50.243+0000] {dag.py:3211} INFO - Sync 1 DAGs
[2024-09-09T10:19:50.254+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:19:50.254+0000] {dag.py:3234} INFO - Creating ORM DAG for weatherbit_dag
[2024-09-09T10:19:50.260+0000] {processor.py:211} ERROR - Got an exception! Propagating...
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/dag_processing/processor.py", line 207, in _run_file_processor
    _handle_dag_file_processing()
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/dag_processing/processor.py", line 188, in _handle_dag_file_processing
    result: tuple[int, int, int] = dag_file_processor.process_file(
                                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/session.py", line 97, in wrapper
    return func(*args, session=session, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/dag_processing/processor.py", line 942, in process_file
    serialize_errors = DagFileProcessor.save_dag_to_db(
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/api_internal/internal_api_call.py", line 139, in wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/session.py", line 97, in wrapper
    return func(*args, session=session, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/dag_processing/processor.py", line 982, in save_dag_to_db
    import_errors = DagBag._sync_to_db(dags=dags, processor_subdir=dag_directory, session=session)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/session.py", line 94, in wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 707, in _sync_to_db
    for attempt in run_with_db_retries(logger=log):
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/tenacity/__init__.py", line 443, in __iter__
    do = self.iter(retry_state=retry_state)
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/tenacity/__init__.py", line 376, in iter
    result = action(retry_state)
             ^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/tenacity/__init__.py", line 398, in <lambda>
    self._add_action_func(lambda rs: rs.outcome.result())
                                     ^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/concurrent/futures/_base.py", line 449, in result
    return self.__get_result()
           ^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/concurrent/futures/_base.py", line 401, in __get_result
    raise self._exception
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 723, in _sync_to_db
    DAG.bulk_write_to_db(dags.values(), processor_subdir=processor_subdir, session=session)
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/session.py", line 94, in wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dag.py", line 3290, in bulk_write_to_db
    orm_dag.calculate_dagrun_date_fields(dag, last_automated_data_interval)
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dag.py", line 4130, in calculate_dagrun_date_fields
    next_dagrun_info = dag.next_dagrun_info(last_automated_data_interval)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dag.py", line 1155, in next_dagrun_info
    restriction = self._time_restriction
                  ^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/functools.py", line 993, in __get__
    val = self.func(instance)
          ^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dag.py", line 1194, in _time_restriction
    earliest = timezone.coerce_datetime(min(start_dates))
                                        ^^^^^^^^^^^^^^^^
TypeError: '<' not supported between instances of 'str' and 'DateTime'
[2024-09-09T10:22:43.451+0000] {processor.py:186} INFO - Started process (PID=54) to work on /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:22:43.452+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/weatherbit_dag.py for tasks to queue
[2024-09-09T10:22:43.453+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:22:43.453+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:22:43.484+0000] {processor.py:925} INFO - DAG(s) 'weatherbit_dag' retrieved from /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:22:43.648+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:22:43.647+0000] {override.py:1858} INFO - Created Permission View: can read on DAG:weatherbit_dag
[2024-09-09T10:22:43.656+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:22:43.656+0000] {override.py:1858} INFO - Created Permission View: can delete on DAG:weatherbit_dag
[2024-09-09T10:22:43.661+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:22:43.661+0000] {override.py:1858} INFO - Created Permission View: can edit on DAG:weatherbit_dag
[2024-09-09T10:22:43.662+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:22:43.662+0000] {dag.py:3211} INFO - Sync 1 DAGs
[2024-09-09T10:22:43.672+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:22:43.671+0000] {dag.py:3234} INFO - Creating ORM DAG for weatherbit_dag
[2024-09-09T10:22:43.679+0000] {processor.py:211} ERROR - Got an exception! Propagating...
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/dag_processing/processor.py", line 207, in _run_file_processor
    _handle_dag_file_processing()
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/dag_processing/processor.py", line 188, in _handle_dag_file_processing
    result: tuple[int, int, int] = dag_file_processor.process_file(
                                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/session.py", line 97, in wrapper
    return func(*args, session=session, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/dag_processing/processor.py", line 942, in process_file
    serialize_errors = DagFileProcessor.save_dag_to_db(
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/api_internal/internal_api_call.py", line 139, in wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/session.py", line 97, in wrapper
    return func(*args, session=session, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/dag_processing/processor.py", line 982, in save_dag_to_db
    import_errors = DagBag._sync_to_db(dags=dags, processor_subdir=dag_directory, session=session)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/session.py", line 94, in wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 707, in _sync_to_db
    for attempt in run_with_db_retries(logger=log):
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/tenacity/__init__.py", line 443, in __iter__
    do = self.iter(retry_state=retry_state)
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/tenacity/__init__.py", line 376, in iter
    result = action(retry_state)
             ^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/tenacity/__init__.py", line 398, in <lambda>
    self._add_action_func(lambda rs: rs.outcome.result())
                                     ^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/concurrent/futures/_base.py", line 449, in result
    return self.__get_result()
           ^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/concurrent/futures/_base.py", line 401, in __get_result
    raise self._exception
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 723, in _sync_to_db
    DAG.bulk_write_to_db(dags.values(), processor_subdir=processor_subdir, session=session)
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/session.py", line 94, in wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dag.py", line 3290, in bulk_write_to_db
    orm_dag.calculate_dagrun_date_fields(dag, last_automated_data_interval)
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dag.py", line 4130, in calculate_dagrun_date_fields
    next_dagrun_info = dag.next_dagrun_info(last_automated_data_interval)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dag.py", line 1155, in next_dagrun_info
    restriction = self._time_restriction
                  ^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/functools.py", line 993, in __get__
    val = self.func(instance)
          ^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dag.py", line 1194, in _time_restriction
    earliest = timezone.coerce_datetime(min(start_dates))
                                        ^^^^^^^^^^^^^^^^
TypeError: '<' not supported between instances of 'str' and 'DateTime'
[2024-09-09T10:23:13.829+0000] {processor.py:186} INFO - Started process (PID=61) to work on /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:23:13.830+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/weatherbit_dag.py for tasks to queue
[2024-09-09T10:23:13.831+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:23:13.831+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:23:13.856+0000] {processor.py:925} INFO - DAG(s) 'weatherbit_dag' retrieved from /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:23:13.874+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:23:13.874+0000] {dag.py:3211} INFO - Sync 1 DAGs
[2024-09-09T10:23:13.887+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:23:13.887+0000] {dag.py:3234} INFO - Creating ORM DAG for weatherbit_dag
[2024-09-09T10:23:13.893+0000] {processor.py:211} ERROR - Got an exception! Propagating...
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/dag_processing/processor.py", line 207, in _run_file_processor
    _handle_dag_file_processing()
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/dag_processing/processor.py", line 188, in _handle_dag_file_processing
    result: tuple[int, int, int] = dag_file_processor.process_file(
                                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/session.py", line 97, in wrapper
    return func(*args, session=session, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/dag_processing/processor.py", line 942, in process_file
    serialize_errors = DagFileProcessor.save_dag_to_db(
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/api_internal/internal_api_call.py", line 139, in wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/session.py", line 97, in wrapper
    return func(*args, session=session, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/dag_processing/processor.py", line 982, in save_dag_to_db
    import_errors = DagBag._sync_to_db(dags=dags, processor_subdir=dag_directory, session=session)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/session.py", line 94, in wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 707, in _sync_to_db
    for attempt in run_with_db_retries(logger=log):
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/tenacity/__init__.py", line 443, in __iter__
    do = self.iter(retry_state=retry_state)
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/tenacity/__init__.py", line 376, in iter
    result = action(retry_state)
             ^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/tenacity/__init__.py", line 398, in <lambda>
    self._add_action_func(lambda rs: rs.outcome.result())
                                     ^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/concurrent/futures/_base.py", line 449, in result
    return self.__get_result()
           ^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/concurrent/futures/_base.py", line 401, in __get_result
    raise self._exception
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 723, in _sync_to_db
    DAG.bulk_write_to_db(dags.values(), processor_subdir=processor_subdir, session=session)
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/session.py", line 94, in wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dag.py", line 3290, in bulk_write_to_db
    orm_dag.calculate_dagrun_date_fields(dag, last_automated_data_interval)
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dag.py", line 4130, in calculate_dagrun_date_fields
    next_dagrun_info = dag.next_dagrun_info(last_automated_data_interval)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dag.py", line 1155, in next_dagrun_info
    restriction = self._time_restriction
                  ^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/functools.py", line 993, in __get__
    val = self.func(instance)
          ^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dag.py", line 1194, in _time_restriction
    earliest = timezone.coerce_datetime(min(start_dates))
                                        ^^^^^^^^^^^^^^^^
TypeError: '<' not supported between instances of 'str' and 'DateTime'
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      [2024-09-09T10:25:05.707+0000] {processor.py:186} INFO - Started process (PID=47) to work on /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:25:05.707+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/weatherbit_dag.py for tasks to queue
[2024-09-09T10:25:05.709+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:25:05.709+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:25:05.734+0000] {processor.py:925} INFO - DAG(s) 'weatherbit_dag' retrieved from /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:25:05.754+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:25:05.754+0000] {dag.py:3211} INFO - Sync 1 DAGs
[2024-09-09T10:25:05.764+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:25:05.764+0000] {dag.py:3234} INFO - Creating ORM DAG for weatherbit_dag
[2024-09-09T10:25:05.772+0000] {processor.py:211} ERROR - Got an exception! Propagating...
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/dag_processing/processor.py", line 207, in _run_file_processor
    _handle_dag_file_processing()
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/dag_processing/processor.py", line 188, in _handle_dag_file_processing
    result: tuple[int, int, int] = dag_file_processor.process_file(
                                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/session.py", line 97, in wrapper
    return func(*args, session=session, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/dag_processing/processor.py", line 942, in process_file
    serialize_errors = DagFileProcessor.save_dag_to_db(
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/api_internal/internal_api_call.py", line 139, in wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/session.py", line 97, in wrapper
    return func(*args, session=session, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/dag_processing/processor.py", line 982, in save_dag_to_db
    import_errors = DagBag._sync_to_db(dags=dags, processor_subdir=dag_directory, session=session)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/session.py", line 94, in wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 707, in _sync_to_db
    for attempt in run_with_db_retries(logger=log):
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/tenacity/__init__.py", line 443, in __iter__
    do = self.iter(retry_state=retry_state)
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/tenacity/__init__.py", line 376, in iter
    result = action(retry_state)
             ^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/tenacity/__init__.py", line 398, in <lambda>
    self._add_action_func(lambda rs: rs.outcome.result())
                                     ^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/concurrent/futures/_base.py", line 449, in result
    return self.__get_result()
           ^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/concurrent/futures/_base.py", line 401, in __get_result
    raise self._exception
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 723, in _sync_to_db
    DAG.bulk_write_to_db(dags.values(), processor_subdir=processor_subdir, session=session)
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/session.py", line 94, in wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dag.py", line 3290, in bulk_write_to_db
    orm_dag.calculate_dagrun_date_fields(dag, last_automated_data_interval)
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dag.py", line 4130, in calculate_dagrun_date_fields
    next_dagrun_info = dag.next_dagrun_info(last_automated_data_interval)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dag.py", line 1155, in next_dagrun_info
    restriction = self._time_restriction
                  ^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/functools.py", line 993, in __get__
    val = self.func(instance)
          ^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dag.py", line 1194, in _time_restriction
    earliest = timezone.coerce_datetime(min(start_dates))
                                        ^^^^^^^^^^^^^^^^
TypeError: '<' not supported between instances of 'str' and 'DateTime'
[2024-09-09T10:27:56.912+0000] {processor.py:186} INFO - Started process (PID=54) to work on /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:27:56.912+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/weatherbit_dag.py for tasks to queue
[2024-09-09T10:27:56.914+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:27:56.914+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:27:56.942+0000] {processor.py:925} INFO - DAG(s) 'weatherbit_dag' retrieved from /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:27:57.115+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:27:57.114+0000] {override.py:1858} INFO - Created Permission View: can read on DAG:weatherbit_dag
[2024-09-09T10:27:57.123+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:27:57.123+0000] {override.py:1858} INFO - Created Permission View: can edit on DAG:weatherbit_dag
[2024-09-09T10:27:57.129+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:27:57.128+0000] {override.py:1858} INFO - Created Permission View: can delete on DAG:weatherbit_dag
[2024-09-09T10:27:57.129+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:27:57.129+0000] {dag.py:3211} INFO - Sync 1 DAGs
[2024-09-09T10:27:57.142+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:27:57.141+0000] {dag.py:3234} INFO - Creating ORM DAG for weatherbit_dag
[2024-09-09T10:27:57.150+0000] {processor.py:211} ERROR - Got an exception! Propagating...
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/dag_processing/processor.py", line 207, in _run_file_processor
    _handle_dag_file_processing()
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/dag_processing/processor.py", line 188, in _handle_dag_file_processing
    result: tuple[int, int, int] = dag_file_processor.process_file(
                                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/session.py", line 97, in wrapper
    return func(*args, session=session, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/dag_processing/processor.py", line 942, in process_file
    serialize_errors = DagFileProcessor.save_dag_to_db(
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/api_internal/internal_api_call.py", line 139, in wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/session.py", line 97, in wrapper
    return func(*args, session=session, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/dag_processing/processor.py", line 982, in save_dag_to_db
    import_errors = DagBag._sync_to_db(dags=dags, processor_subdir=dag_directory, session=session)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/session.py", line 94, in wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 707, in _sync_to_db
    for attempt in run_with_db_retries(logger=log):
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/tenacity/__init__.py", line 443, in __iter__
    do = self.iter(retry_state=retry_state)
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/tenacity/__init__.py", line 376, in iter
    result = action(retry_state)
             ^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/tenacity/__init__.py", line 398, in <lambda>
    self._add_action_func(lambda rs: rs.outcome.result())
                                     ^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/concurrent/futures/_base.py", line 449, in result
    return self.__get_result()
           ^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/concurrent/futures/_base.py", line 401, in __get_result
    raise self._exception
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 723, in _sync_to_db
    DAG.bulk_write_to_db(dags.values(), processor_subdir=processor_subdir, session=session)
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/session.py", line 94, in wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dag.py", line 3290, in bulk_write_to_db
    orm_dag.calculate_dagrun_date_fields(dag, last_automated_data_interval)
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dag.py", line 4130, in calculate_dagrun_date_fields
    next_dagrun_info = dag.next_dagrun_info(last_automated_data_interval)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dag.py", line 1155, in next_dagrun_info
    restriction = self._time_restriction
                  ^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/functools.py", line 993, in __get__
    val = self.func(instance)
          ^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dag.py", line 1194, in _time_restriction
    earliest = timezone.coerce_datetime(min(start_dates))
                                        ^^^^^^^^^^^^^^^^
TypeError: '<' not supported between instances of 'str' and 'DateTime'
[2024-09-09T10:28:27.300+0000] {processor.py:186} INFO - Started process (PID=61) to work on /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:28:27.300+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/weatherbit_dag.py for tasks to queue
[2024-09-09T10:28:27.302+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:28:27.302+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:28:27.326+0000] {processor.py:925} INFO - DAG(s) 'weatherbit_dag' retrieved from /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:28:27.343+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:28:27.343+0000] {dag.py:3211} INFO - Sync 1 DAGs
[2024-09-09T10:28:27.354+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:28:27.354+0000] {dag.py:3234} INFO - Creating ORM DAG for weatherbit_dag
[2024-09-09T10:28:27.361+0000] {processor.py:211} ERROR - Got an exception! Propagating...
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/dag_processing/processor.py", line 207, in _run_file_processor
    _handle_dag_file_processing()
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/dag_processing/processor.py", line 188, in _handle_dag_file_processing
    result: tuple[int, int, int] = dag_file_processor.process_file(
                                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/session.py", line 97, in wrapper
    return func(*args, session=session, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/dag_processing/processor.py", line 942, in process_file
    serialize_errors = DagFileProcessor.save_dag_to_db(
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/api_internal/internal_api_call.py", line 139, in wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/session.py", line 97, in wrapper
    return func(*args, session=session, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/dag_processing/processor.py", line 982, in save_dag_to_db
    import_errors = DagBag._sync_to_db(dags=dags, processor_subdir=dag_directory, session=session)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/session.py", line 94, in wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 707, in _sync_to_db
    for attempt in run_with_db_retries(logger=log):
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/tenacity/__init__.py", line 443, in __iter__
    do = self.iter(retry_state=retry_state)
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/tenacity/__init__.py", line 376, in iter
    result = action(retry_state)
             ^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/tenacity/__init__.py", line 398, in <lambda>
    self._add_action_func(lambda rs: rs.outcome.result())
                                     ^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/concurrent/futures/_base.py", line 449, in result
    return self.__get_result()
           ^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/concurrent/futures/_base.py", line 401, in __get_result
    raise self._exception
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 723, in _sync_to_db
    DAG.bulk_write_to_db(dags.values(), processor_subdir=processor_subdir, session=session)
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/session.py", line 94, in wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dag.py", line 3290, in bulk_write_to_db
    orm_dag.calculate_dagrun_date_fields(dag, last_automated_data_interval)
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dag.py", line 4130, in calculate_dagrun_date_fields
    next_dagrun_info = dag.next_dagrun_info(last_automated_data_interval)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dag.py", line 1155, in next_dagrun_info
    restriction = self._time_restriction
                  ^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/functools.py", line 993, in __get__
    val = self.func(instance)
          ^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dag.py", line 1194, in _time_restriction
    earliest = timezone.coerce_datetime(min(start_dates))
                                        ^^^^^^^^^^^^^^^^
TypeError: '<' not supported between instances of 'str' and 'DateTime'
[2024-09-09T10:28:57.440+0000] {processor.py:186} INFO - Started process (PID=68) to work on /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:28:57.440+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/weatherbit_dag.py for tasks to queue
[2024-09-09T10:28:57.442+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:28:57.442+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:28:57.466+0000] {processor.py:925} INFO - DAG(s) 'weatherbit_dag' retrieved from /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:28:57.487+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:28:57.487+0000] {dag.py:3211} INFO - Sync 1 DAGs
[2024-09-09T10:28:57.498+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:28:57.498+0000] {dag.py:3234} INFO - Creating ORM DAG for weatherbit_dag
[2024-09-09T10:28:57.505+0000] {processor.py:211} ERROR - Got an exception! Propagating...
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/dag_processing/processor.py", line 207, in _run_file_processor
    _handle_dag_file_processing()
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/dag_processing/processor.py", line 188, in _handle_dag_file_processing
    result: tuple[int, int, int] = dag_file_processor.process_file(
                                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/session.py", line 97, in wrapper
    return func(*args, session=session, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/dag_processing/processor.py", line 942, in process_file
    serialize_errors = DagFileProcessor.save_dag_to_db(
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/api_internal/internal_api_call.py", line 139, in wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/session.py", line 97, in wrapper
    return func(*args, session=session, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/dag_processing/processor.py", line 982, in save_dag_to_db
    import_errors = DagBag._sync_to_db(dags=dags, processor_subdir=dag_directory, session=session)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/session.py", line 94, in wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 707, in _sync_to_db
    for attempt in run_with_db_retries(logger=log):
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/tenacity/__init__.py", line 443, in __iter__
    do = self.iter(retry_state=retry_state)
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/tenacity/__init__.py", line 376, in iter
    result = action(retry_state)
             ^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/tenacity/__init__.py", line 398, in <lambda>
    self._add_action_func(lambda rs: rs.outcome.result())
                                     ^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/concurrent/futures/_base.py", line 449, in result
    return self.__get_result()
           ^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/concurrent/futures/_base.py", line 401, in __get_result
    raise self._exception
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 723, in _sync_to_db
    DAG.bulk_write_to_db(dags.values(), processor_subdir=processor_subdir, session=session)
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/session.py", line 94, in wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dag.py", line 3290, in bulk_write_to_db
    orm_dag.calculate_dagrun_date_fields(dag, last_automated_data_interval)
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dag.py", line 4130, in calculate_dagrun_date_fields
    next_dagrun_info = dag.next_dagrun_info(last_automated_data_interval)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dag.py", line 1155, in next_dagrun_info
    restriction = self._time_restriction
                  ^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/functools.py", line 993, in __get__
    val = self.func(instance)
          ^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dag.py", line 1194, in _time_restriction
    earliest = timezone.coerce_datetime(min(start_dates))
                                        ^^^^^^^^^^^^^^^^
TypeError: '<' not supported between instances of 'str' and 'DateTime'
[2024-09-09T10:29:27.632+0000] {processor.py:186} INFO - Started process (PID=76) to work on /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:29:27.633+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/weatherbit_dag.py for tasks to queue
[2024-09-09T10:29:27.634+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:29:27.634+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:29:27.658+0000] {processor.py:925} INFO - DAG(s) 'weatherbit_dag' retrieved from /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:29:27.674+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:29:27.674+0000] {dag.py:3211} INFO - Sync 1 DAGs
[2024-09-09T10:29:27.684+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:29:27.684+0000] {dag.py:3234} INFO - Creating ORM DAG for weatherbit_dag
[2024-09-09T10:29:27.690+0000] {processor.py:211} ERROR - Got an exception! Propagating...
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/dag_processing/processor.py", line 207, in _run_file_processor
    _handle_dag_file_processing()
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/dag_processing/processor.py", line 188, in _handle_dag_file_processing
    result: tuple[int, int, int] = dag_file_processor.process_file(
                                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/session.py", line 97, in wrapper
    return func(*args, session=session, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/dag_processing/processor.py", line 942, in process_file
    serialize_errors = DagFileProcessor.save_dag_to_db(
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/api_internal/internal_api_call.py", line 139, in wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/session.py", line 97, in wrapper
    return func(*args, session=session, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/dag_processing/processor.py", line 982, in save_dag_to_db
    import_errors = DagBag._sync_to_db(dags=dags, processor_subdir=dag_directory, session=session)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/session.py", line 94, in wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 707, in _sync_to_db
    for attempt in run_with_db_retries(logger=log):
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/tenacity/__init__.py", line 443, in __iter__
    do = self.iter(retry_state=retry_state)
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/tenacity/__init__.py", line 376, in iter
    result = action(retry_state)
             ^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/tenacity/__init__.py", line 398, in <lambda>
    self._add_action_func(lambda rs: rs.outcome.result())
                                     ^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/concurrent/futures/_base.py", line 449, in result
    return self.__get_result()
           ^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/concurrent/futures/_base.py", line 401, in __get_result
    raise self._exception
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 723, in _sync_to_db
    DAG.bulk_write_to_db(dags.values(), processor_subdir=processor_subdir, session=session)
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/session.py", line 94, in wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dag.py", line 3290, in bulk_write_to_db
    orm_dag.calculate_dagrun_date_fields(dag, last_automated_data_interval)
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dag.py", line 4130, in calculate_dagrun_date_fields
    next_dagrun_info = dag.next_dagrun_info(last_automated_data_interval)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dag.py", line 1155, in next_dagrun_info
    restriction = self._time_restriction
                  ^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/functools.py", line 993, in __get__
    val = self.func(instance)
          ^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dag.py", line 1194, in _time_restriction
    earliest = timezone.coerce_datetime(min(start_dates))
                                        ^^^^^^^^^^^^^^^^
TypeError: '<' not supported between instances of 'str' and 'DateTime'
[2024-09-09T10:29:57.781+0000] {processor.py:186} INFO - Started process (PID=83) to work on /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:29:57.782+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/weatherbit_dag.py for tasks to queue
[2024-09-09T10:29:57.783+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:29:57.783+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:29:57.808+0000] {processor.py:925} INFO - DAG(s) 'weatherbit_dag' retrieved from /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:29:57.826+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:29:57.826+0000] {dag.py:3211} INFO - Sync 1 DAGs
[2024-09-09T10:29:57.835+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:29:57.835+0000] {dag.py:3234} INFO - Creating ORM DAG for weatherbit_dag
[2024-09-09T10:29:57.842+0000] {processor.py:211} ERROR - Got an exception! Propagating...
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/dag_processing/processor.py", line 207, in _run_file_processor
    _handle_dag_file_processing()
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/dag_processing/processor.py", line 188, in _handle_dag_file_processing
    result: tuple[int, int, int] = dag_file_processor.process_file(
                                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/session.py", line 97, in wrapper
    return func(*args, session=session, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/dag_processing/processor.py", line 942, in process_file
    serialize_errors = DagFileProcessor.save_dag_to_db(
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/api_internal/internal_api_call.py", line 139, in wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/session.py", line 97, in wrapper
    return func(*args, session=session, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/dag_processing/processor.py", line 982, in save_dag_to_db
    import_errors = DagBag._sync_to_db(dags=dags, processor_subdir=dag_directory, session=session)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/session.py", line 94, in wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 707, in _sync_to_db
    for attempt in run_with_db_retries(logger=log):
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/tenacity/__init__.py", line 443, in __iter__
    do = self.iter(retry_state=retry_state)
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/tenacity/__init__.py", line 376, in iter
    result = action(retry_state)
             ^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/tenacity/__init__.py", line 398, in <lambda>
    self._add_action_func(lambda rs: rs.outcome.result())
                                     ^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/concurrent/futures/_base.py", line 449, in result
    return self.__get_result()
           ^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/concurrent/futures/_base.py", line 401, in __get_result
    raise self._exception
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 723, in _sync_to_db
    DAG.bulk_write_to_db(dags.values(), processor_subdir=processor_subdir, session=session)
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/session.py", line 94, in wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dag.py", line 3290, in bulk_write_to_db
    orm_dag.calculate_dagrun_date_fields(dag, last_automated_data_interval)
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dag.py", line 4130, in calculate_dagrun_date_fields
    next_dagrun_info = dag.next_dagrun_info(last_automated_data_interval)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dag.py", line 1155, in next_dagrun_info
    restriction = self._time_restriction
                  ^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/functools.py", line 993, in __get__
    val = self.func(instance)
          ^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dag.py", line 1194, in _time_restriction
    earliest = timezone.coerce_datetime(min(start_dates))
                                        ^^^^^^^^^^^^^^^^
TypeError: '<' not supported between instances of 'str' and 'DateTime'
[2024-09-09T10:30:27.962+0000] {processor.py:186} INFO - Started process (PID=90) to work on /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:30:27.962+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/weatherbit_dag.py for tasks to queue
[2024-09-09T10:30:27.964+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:30:27.964+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:30:27.987+0000] {processor.py:925} INFO - DAG(s) 'weatherbit_dag' retrieved from /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:30:28.003+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:30:28.003+0000] {dag.py:3211} INFO - Sync 1 DAGs
[2024-09-09T10:30:28.013+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:30:28.013+0000] {dag.py:3234} INFO - Creating ORM DAG for weatherbit_dag
[2024-09-09T10:30:28.019+0000] {processor.py:211} ERROR - Got an exception! Propagating...
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/dag_processing/processor.py", line 207, in _run_file_processor
    _handle_dag_file_processing()
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/dag_processing/processor.py", line 188, in _handle_dag_file_processing
    result: tuple[int, int, int] = dag_file_processor.process_file(
                                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/session.py", line 97, in wrapper
    return func(*args, session=session, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/dag_processing/processor.py", line 942, in process_file
    serialize_errors = DagFileProcessor.save_dag_to_db(
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/api_internal/internal_api_call.py", line 139, in wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/session.py", line 97, in wrapper
    return func(*args, session=session, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/dag_processing/processor.py", line 982, in save_dag_to_db
    import_errors = DagBag._sync_to_db(dags=dags, processor_subdir=dag_directory, session=session)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/session.py", line 94, in wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 707, in _sync_to_db
    for attempt in run_with_db_retries(logger=log):
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/tenacity/__init__.py", line 443, in __iter__
    do = self.iter(retry_state=retry_state)
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/tenacity/__init__.py", line 376, in iter
    result = action(retry_state)
             ^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/tenacity/__init__.py", line 398, in <lambda>
    self._add_action_func(lambda rs: rs.outcome.result())
                                     ^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/concurrent/futures/_base.py", line 449, in result
    return self.__get_result()
           ^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/concurrent/futures/_base.py", line 401, in __get_result
    raise self._exception
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 723, in _sync_to_db
    DAG.bulk_write_to_db(dags.values(), processor_subdir=processor_subdir, session=session)
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/session.py", line 94, in wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dag.py", line 3290, in bulk_write_to_db
    orm_dag.calculate_dagrun_date_fields(dag, last_automated_data_interval)
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dag.py", line 4130, in calculate_dagrun_date_fields
    next_dagrun_info = dag.next_dagrun_info(last_automated_data_interval)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dag.py", line 1155, in next_dagrun_info
    restriction = self._time_restriction
                  ^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/functools.py", line 993, in __get__
    val = self.func(instance)
          ^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dag.py", line 1194, in _time_restriction
    earliest = timezone.coerce_datetime(min(start_dates))
                                        ^^^^^^^^^^^^^^^^
TypeError: '<' not supported between instances of 'str' and 'DateTime'
[2024-09-09T10:30:58.114+0000] {processor.py:186} INFO - Started process (PID=97) to work on /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:30:58.114+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/weatherbit_dag.py for tasks to queue
[2024-09-09T10:30:58.116+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:30:58.116+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:30:58.140+0000] {processor.py:925} INFO - DAG(s) 'weatherbit_dag' retrieved from /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:30:58.156+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:30:58.155+0000] {dag.py:3211} INFO - Sync 1 DAGs
[2024-09-09T10:30:58.166+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:30:58.166+0000] {dag.py:3234} INFO - Creating ORM DAG for weatherbit_dag
[2024-09-09T10:30:58.173+0000] {processor.py:211} ERROR - Got an exception! Propagating...
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/dag_processing/processor.py", line 207, in _run_file_processor
    _handle_dag_file_processing()
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/dag_processing/processor.py", line 188, in _handle_dag_file_processing
    result: tuple[int, int, int] = dag_file_processor.process_file(
                                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/session.py", line 97, in wrapper
    return func(*args, session=session, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/dag_processing/processor.py", line 942, in process_file
    serialize_errors = DagFileProcessor.save_dag_to_db(
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/api_internal/internal_api_call.py", line 139, in wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/session.py", line 97, in wrapper
    return func(*args, session=session, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/dag_processing/processor.py", line 982, in save_dag_to_db
    import_errors = DagBag._sync_to_db(dags=dags, processor_subdir=dag_directory, session=session)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/session.py", line 94, in wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 707, in _sync_to_db
    for attempt in run_with_db_retries(logger=log):
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/tenacity/__init__.py", line 443, in __iter__
    do = self.iter(retry_state=retry_state)
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/tenacity/__init__.py", line 376, in iter
    result = action(retry_state)
             ^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/tenacity/__init__.py", line 398, in <lambda>
    self._add_action_func(lambda rs: rs.outcome.result())
                                     ^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/concurrent/futures/_base.py", line 449, in result
    return self.__get_result()
           ^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/concurrent/futures/_base.py", line 401, in __get_result
    raise self._exception
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 723, in _sync_to_db
    DAG.bulk_write_to_db(dags.values(), processor_subdir=processor_subdir, session=session)
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/session.py", line 94, in wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dag.py", line 3290, in bulk_write_to_db
    orm_dag.calculate_dagrun_date_fields(dag, last_automated_data_interval)
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dag.py", line 4130, in calculate_dagrun_date_fields
    next_dagrun_info = dag.next_dagrun_info(last_automated_data_interval)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dag.py", line 1155, in next_dagrun_info
    restriction = self._time_restriction
                  ^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/functools.py", line 993, in __get__
    val = self.func(instance)
          ^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dag.py", line 1194, in _time_restriction
    earliest = timezone.coerce_datetime(min(start_dates))
                                        ^^^^^^^^^^^^^^^^
TypeError: '<' not supported between instances of 'str' and 'DateTime'
[2024-09-09T10:31:16.258+0000] {processor.py:186} INFO - Started process (PID=104) to work on /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:31:16.259+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/weatherbit_dag.py for tasks to queue
[2024-09-09T10:31:16.260+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:31:16.260+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:31:16.283+0000] {processor.py:925} INFO - DAG(s) 'weatherbit_dag' retrieved from /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:31:16.360+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:31:16.359+0000] {dag.py:3211} INFO - Sync 1 DAGs
[2024-09-09T10:31:16.369+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:31:16.369+0000] {dag.py:3234} INFO - Creating ORM DAG for weatherbit_dag
[2024-09-09T10:31:16.377+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:31:16.377+0000] {dag.py:4138} INFO - Setting next_dagrun for weatherbit_dag to 2024-09-08 00:00:00+00:00, run_after=2024-09-09 00:00:00+00:00
[2024-09-09T10:31:16.391+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/weatherbit_dag.py took 0.137 seconds
[2024-09-09T10:31:46.531+0000] {processor.py:186} INFO - Started process (PID=111) to work on /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:31:46.532+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/weatherbit_dag.py for tasks to queue
[2024-09-09T10:31:46.533+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:31:46.533+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:31:46.548+0000] {processor.py:925} INFO - DAG(s) 'weatherbit_dag' retrieved from /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:31:46.570+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:31:46.570+0000] {dag.py:3211} INFO - Sync 1 DAGs
[2024-09-09T10:31:46.587+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:31:46.587+0000] {dag.py:4138} INFO - Setting next_dagrun for weatherbit_dag to 2024-09-08 00:00:00+00:00, run_after=2024-09-09 00:00:00+00:00
[2024-09-09T10:31:46.601+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/weatherbit_dag.py took 0.074 seconds
[2024-09-09T10:32:16.674+0000] {processor.py:186} INFO - Started process (PID=118) to work on /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:32:16.675+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/weatherbit_dag.py for tasks to queue
[2024-09-09T10:32:16.677+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:32:16.676+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:32:16.691+0000] {processor.py:925} INFO - DAG(s) 'weatherbit_dag' retrieved from /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:32:16.708+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:32:16.708+0000] {dag.py:3211} INFO - Sync 1 DAGs
[2024-09-09T10:32:16.727+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:32:16.727+0000] {dag.py:4138} INFO - Setting next_dagrun for weatherbit_dag to 2024-09-08 00:00:00+00:00, run_after=2024-09-09 00:00:00+00:00
[2024-09-09T10:32:16.740+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/weatherbit_dag.py took 0.069 seconds
[2024-09-09T10:32:46.878+0000] {processor.py:186} INFO - Started process (PID=125) to work on /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:32:46.879+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/weatherbit_dag.py for tasks to queue
[2024-09-09T10:32:46.880+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:32:46.880+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:32:46.897+0000] {processor.py:925} INFO - DAG(s) 'weatherbit_dag' retrieved from /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:32:46.918+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:32:46.918+0000] {dag.py:3211} INFO - Sync 1 DAGs
[2024-09-09T10:32:46.940+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:32:46.940+0000] {dag.py:4138} INFO - Setting next_dagrun for weatherbit_dag to 2024-09-08 00:00:00+00:00, run_after=2024-09-09 00:00:00+00:00
[2024-09-09T10:32:46.964+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/weatherbit_dag.py took 0.089 seconds
[2024-09-09T10:33:17.063+0000] {processor.py:186} INFO - Started process (PID=136) to work on /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:33:17.064+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/weatherbit_dag.py for tasks to queue
[2024-09-09T10:33:17.065+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:33:17.065+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:33:17.081+0000] {processor.py:925} INFO - DAG(s) 'weatherbit_dag' retrieved from /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:33:17.098+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:33:17.098+0000] {dag.py:3211} INFO - Sync 1 DAGs
[2024-09-09T10:33:17.116+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:33:17.115+0000] {dag.py:4138} INFO - Setting next_dagrun for weatherbit_dag to 2024-09-09 00:00:00+00:00, run_after=2024-09-10 00:00:00+00:00
[2024-09-09T10:33:17.129+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/weatherbit_dag.py took 0.071 seconds
[2024-09-09T10:33:47.239+0000] {processor.py:186} INFO - Started process (PID=143) to work on /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:33:47.240+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/weatherbit_dag.py for tasks to queue
[2024-09-09T10:33:47.242+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:33:47.242+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:33:47.259+0000] {processor.py:925} INFO - DAG(s) 'weatherbit_dag' retrieved from /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:33:47.277+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:33:47.277+0000] {dag.py:3211} INFO - Sync 1 DAGs
[2024-09-09T10:33:47.296+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:33:47.295+0000] {dag.py:4138} INFO - Setting next_dagrun for weatherbit_dag to 2024-09-09 00:00:00+00:00, run_after=2024-09-10 00:00:00+00:00
[2024-09-09T10:33:47.309+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/weatherbit_dag.py took 0.074 seconds
[2024-09-09T10:34:17.390+0000] {processor.py:186} INFO - Started process (PID=150) to work on /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:34:17.391+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/weatherbit_dag.py for tasks to queue
[2024-09-09T10:34:17.392+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:34:17.392+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:34:17.407+0000] {processor.py:925} INFO - DAG(s) 'weatherbit_dag' retrieved from /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:34:17.424+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:34:17.424+0000] {dag.py:3211} INFO - Sync 1 DAGs
[2024-09-09T10:34:17.441+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:34:17.441+0000] {dag.py:4138} INFO - Setting next_dagrun for weatherbit_dag to 2024-09-09 00:00:00+00:00, run_after=2024-09-10 00:00:00+00:00
[2024-09-09T10:34:17.454+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/weatherbit_dag.py took 0.067 seconds
[2024-09-09T10:34:47.583+0000] {processor.py:186} INFO - Started process (PID=157) to work on /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:34:47.583+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/weatherbit_dag.py for tasks to queue
[2024-09-09T10:34:47.585+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:34:47.584+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:34:47.600+0000] {processor.py:925} INFO - DAG(s) 'weatherbit_dag' retrieved from /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:34:47.618+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:34:47.618+0000] {dag.py:3211} INFO - Sync 1 DAGs
[2024-09-09T10:34:47.635+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:34:47.635+0000] {dag.py:4138} INFO - Setting next_dagrun for weatherbit_dag to 2024-09-09 00:00:00+00:00, run_after=2024-09-10 00:00:00+00:00
[2024-09-09T10:34:47.649+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/weatherbit_dag.py took 0.070 seconds
[2024-09-09T10:35:17.734+0000] {processor.py:186} INFO - Started process (PID=164) to work on /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:35:17.734+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/weatherbit_dag.py for tasks to queue
[2024-09-09T10:35:17.736+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:35:17.736+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:35:17.752+0000] {processor.py:925} INFO - DAG(s) 'weatherbit_dag' retrieved from /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:35:17.770+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:35:17.770+0000] {dag.py:3211} INFO - Sync 1 DAGs
[2024-09-09T10:35:17.788+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:35:17.788+0000] {dag.py:4138} INFO - Setting next_dagrun for weatherbit_dag to 2024-09-09 00:00:00+00:00, run_after=2024-09-10 00:00:00+00:00
[2024-09-09T10:35:17.803+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/weatherbit_dag.py took 0.073 seconds
[2024-09-09T10:35:47.924+0000] {processor.py:186} INFO - Started process (PID=171) to work on /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:35:47.924+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/weatherbit_dag.py for tasks to queue
[2024-09-09T10:35:47.926+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:35:47.925+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:35:47.942+0000] {processor.py:925} INFO - DAG(s) 'weatherbit_dag' retrieved from /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:35:47.959+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:35:47.959+0000] {dag.py:3211} INFO - Sync 1 DAGs
[2024-09-09T10:35:47.976+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:35:47.976+0000] {dag.py:4138} INFO - Setting next_dagrun for weatherbit_dag to 2024-09-09 00:00:00+00:00, run_after=2024-09-10 00:00:00+00:00
[2024-09-09T10:35:47.990+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/weatherbit_dag.py took 0.070 seconds
[2024-09-09T10:36:18.073+0000] {processor.py:186} INFO - Started process (PID=178) to work on /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:36:18.074+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/weatherbit_dag.py for tasks to queue
[2024-09-09T10:36:18.076+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:36:18.075+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:36:18.092+0000] {processor.py:925} INFO - DAG(s) 'weatherbit_dag' retrieved from /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:36:18.111+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:36:18.111+0000] {dag.py:3211} INFO - Sync 1 DAGs
[2024-09-09T10:36:18.129+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:36:18.129+0000] {dag.py:4138} INFO - Setting next_dagrun for weatherbit_dag to 2024-09-09 00:00:00+00:00, run_after=2024-09-10 00:00:00+00:00
[2024-09-09T10:36:18.143+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/weatherbit_dag.py took 0.073 seconds
[2024-09-09T10:36:48.268+0000] {processor.py:186} INFO - Started process (PID=185) to work on /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:36:48.269+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/weatherbit_dag.py for tasks to queue
[2024-09-09T10:36:48.270+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:36:48.270+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:36:48.285+0000] {processor.py:925} INFO - DAG(s) 'weatherbit_dag' retrieved from /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:36:48.304+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:36:48.304+0000] {dag.py:3211} INFO - Sync 1 DAGs
[2024-09-09T10:36:48.324+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:36:48.324+0000] {dag.py:4138} INFO - Setting next_dagrun for weatherbit_dag to 2024-09-09 00:00:00+00:00, run_after=2024-09-10 00:00:00+00:00
[2024-09-09T10:36:48.338+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/weatherbit_dag.py took 0.074 seconds
[2024-09-09T10:37:14.409+0000] {processor.py:186} INFO - Started process (PID=192) to work on /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:37:14.409+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/weatherbit_dag.py for tasks to queue
[2024-09-09T10:37:14.411+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:37:14.411+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:37:14.455+0000] {processor.py:925} INFO - DAG(s) 'weatherbit_dag' retrieved from /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:37:14.535+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:37:14.535+0000] {dag.py:3211} INFO - Sync 1 DAGs
[2024-09-09T10:37:14.552+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:37:14.552+0000] {dag.py:4138} INFO - Setting next_dagrun for weatherbit_dag to 2024-09-09 00:00:00+00:00, run_after=2024-09-10 00:00:00+00:00
[2024-09-09T10:37:14.567+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/weatherbit_dag.py took 0.162 seconds
[2024-09-09T10:37:44.597+0000] {processor.py:186} INFO - Started process (PID=199) to work on /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:37:44.598+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/weatherbit_dag.py for tasks to queue
[2024-09-09T10:37:44.599+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:37:44.599+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:37:44.621+0000] {processor.py:925} INFO - DAG(s) 'weatherbit_dag' retrieved from /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:37:44.638+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:37:44.638+0000] {dag.py:3211} INFO - Sync 1 DAGs
[2024-09-09T10:37:44.656+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:37:44.656+0000] {dag.py:4138} INFO - Setting next_dagrun for weatherbit_dag to 2024-09-09 00:00:00+00:00, run_after=2024-09-10 00:00:00+00:00
[2024-09-09T10:37:44.670+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/weatherbit_dag.py took 0.076 seconds
[2024-09-09T10:38:14.776+0000] {processor.py:186} INFO - Started process (PID=206) to work on /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:38:14.777+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/weatherbit_dag.py for tasks to queue
[2024-09-09T10:38:14.778+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:38:14.778+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:38:14.804+0000] {processor.py:925} INFO - DAG(s) 'weatherbit_dag' retrieved from /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:38:14.822+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:38:14.822+0000] {dag.py:3211} INFO - Sync 1 DAGs
[2024-09-09T10:38:14.841+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:38:14.841+0000] {dag.py:4138} INFO - Setting next_dagrun for weatherbit_dag to 2024-09-09 00:00:00+00:00, run_after=2024-09-10 00:00:00+00:00
[2024-09-09T10:38:14.854+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/weatherbit_dag.py took 0.082 seconds
[2024-09-09T10:38:44.935+0000] {processor.py:186} INFO - Started process (PID=217) to work on /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:38:44.936+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/weatherbit_dag.py for tasks to queue
[2024-09-09T10:38:44.938+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:38:44.938+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:38:44.971+0000] {processor.py:925} INFO - DAG(s) 'weatherbit_dag' retrieved from /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:38:44.991+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:38:44.991+0000] {dag.py:3211} INFO - Sync 1 DAGs
[2024-09-09T10:38:45.013+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:38:45.013+0000] {dag.py:4138} INFO - Setting next_dagrun for weatherbit_dag to 2024-09-09 00:00:00+00:00, run_after=2024-09-10 00:00:00+00:00
[2024-09-09T10:38:45.027+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/weatherbit_dag.py took 0.096 seconds
[2024-09-09T10:39:15.132+0000] {processor.py:186} INFO - Started process (PID=224) to work on /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:39:15.132+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/weatherbit_dag.py for tasks to queue
[2024-09-09T10:39:15.134+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:39:15.134+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:39:15.158+0000] {processor.py:925} INFO - DAG(s) 'weatherbit_dag' retrieved from /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:39:15.175+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:39:15.175+0000] {dag.py:3211} INFO - Sync 1 DAGs
[2024-09-09T10:39:15.193+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:39:15.192+0000] {dag.py:4138} INFO - Setting next_dagrun for weatherbit_dag to 2024-09-09 00:00:00+00:00, run_after=2024-09-10 00:00:00+00:00
[2024-09-09T10:39:15.206+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/weatherbit_dag.py took 0.078 seconds
[2024-09-09T10:39:45.305+0000] {processor.py:186} INFO - Started process (PID=231) to work on /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:39:45.306+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/weatherbit_dag.py for tasks to queue
[2024-09-09T10:39:45.308+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:39:45.307+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:39:45.334+0000] {processor.py:925} INFO - DAG(s) 'weatherbit_dag' retrieved from /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:39:45.352+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:39:45.352+0000] {dag.py:3211} INFO - Sync 1 DAGs
[2024-09-09T10:39:45.371+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:39:45.371+0000] {dag.py:4138} INFO - Setting next_dagrun for weatherbit_dag to 2024-09-09 00:00:00+00:00, run_after=2024-09-10 00:00:00+00:00
[2024-09-09T10:39:45.387+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/weatherbit_dag.py took 0.087 seconds
[2024-09-09T10:40:15.490+0000] {processor.py:186} INFO - Started process (PID=238) to work on /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:40:15.490+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/weatherbit_dag.py for tasks to queue
[2024-09-09T10:40:15.492+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:40:15.492+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:40:15.518+0000] {processor.py:925} INFO - DAG(s) 'weatherbit_dag' retrieved from /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:40:15.537+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:40:15.537+0000] {dag.py:3211} INFO - Sync 1 DAGs
[2024-09-09T10:40:15.556+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:40:15.555+0000] {dag.py:4138} INFO - Setting next_dagrun for weatherbit_dag to 2024-09-09 00:00:00+00:00, run_after=2024-09-10 00:00:00+00:00
[2024-09-09T10:40:15.569+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/weatherbit_dag.py took 0.084 seconds
[2024-09-09T10:40:45.670+0000] {processor.py:186} INFO - Started process (PID=245) to work on /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:40:45.671+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/weatherbit_dag.py for tasks to queue
[2024-09-09T10:40:45.672+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:40:45.672+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:40:45.695+0000] {processor.py:925} INFO - DAG(s) 'weatherbit_dag' retrieved from /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:40:45.712+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:40:45.712+0000] {dag.py:3211} INFO - Sync 1 DAGs
[2024-09-09T10:40:45.730+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:40:45.730+0000] {dag.py:4138} INFO - Setting next_dagrun for weatherbit_dag to 2024-09-09 00:00:00+00:00, run_after=2024-09-10 00:00:00+00:00
[2024-09-09T10:40:45.743+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/weatherbit_dag.py took 0.077 seconds
[2024-09-09T10:41:15.842+0000] {processor.py:186} INFO - Started process (PID=252) to work on /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:41:15.843+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/weatherbit_dag.py for tasks to queue
[2024-09-09T10:41:15.845+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:41:15.844+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:41:15.869+0000] {processor.py:925} INFO - DAG(s) 'weatherbit_dag' retrieved from /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:41:15.887+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:41:15.886+0000] {dag.py:3211} INFO - Sync 1 DAGs
[2024-09-09T10:41:15.905+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:41:15.905+0000] {dag.py:4138} INFO - Setting next_dagrun for weatherbit_dag to 2024-09-09 00:00:00+00:00, run_after=2024-09-10 00:00:00+00:00
[2024-09-09T10:41:15.920+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/weatherbit_dag.py took 0.081 seconds
[2024-09-09T10:41:46.017+0000] {processor.py:186} INFO - Started process (PID=259) to work on /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:41:46.017+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/weatherbit_dag.py for tasks to queue
[2024-09-09T10:41:46.019+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:41:46.019+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:41:46.041+0000] {processor.py:925} INFO - DAG(s) 'weatherbit_dag' retrieved from /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:41:46.057+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:41:46.057+0000] {dag.py:3211} INFO - Sync 1 DAGs
[2024-09-09T10:41:46.076+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:41:46.076+0000] {dag.py:4138} INFO - Setting next_dagrun for weatherbit_dag to 2024-09-09 00:00:00+00:00, run_after=2024-09-10 00:00:00+00:00
[2024-09-09T10:41:46.090+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/weatherbit_dag.py took 0.077 seconds
[2024-09-09T10:42:16.200+0000] {processor.py:186} INFO - Started process (PID=266) to work on /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:42:16.200+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/weatherbit_dag.py for tasks to queue
[2024-09-09T10:42:16.202+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:42:16.202+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:42:16.227+0000] {processor.py:925} INFO - DAG(s) 'weatherbit_dag' retrieved from /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:42:16.245+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:42:16.245+0000] {dag.py:3211} INFO - Sync 1 DAGs
[2024-09-09T10:42:16.266+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:42:16.266+0000] {dag.py:4138} INFO - Setting next_dagrun for weatherbit_dag to 2024-09-09 00:00:00+00:00, run_after=2024-09-10 00:00:00+00:00
[2024-09-09T10:42:16.281+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/weatherbit_dag.py took 0.085 seconds
[2024-09-09T10:42:46.363+0000] {processor.py:186} INFO - Started process (PID=273) to work on /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:42:46.363+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/weatherbit_dag.py for tasks to queue
[2024-09-09T10:42:46.365+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:42:46.365+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:42:46.391+0000] {processor.py:925} INFO - DAG(s) 'weatherbit_dag' retrieved from /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:42:46.410+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:42:46.409+0000] {dag.py:3211} INFO - Sync 1 DAGs
[2024-09-09T10:42:46.428+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:42:46.428+0000] {dag.py:4138} INFO - Setting next_dagrun for weatherbit_dag to 2024-09-09 00:00:00+00:00, run_after=2024-09-10 00:00:00+00:00
[2024-09-09T10:42:46.442+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/weatherbit_dag.py took 0.084 seconds
[2024-09-09T10:43:16.591+0000] {processor.py:186} INFO - Started process (PID=280) to work on /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:43:16.592+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/weatherbit_dag.py for tasks to queue
[2024-09-09T10:43:16.593+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:43:16.593+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:43:16.616+0000] {processor.py:925} INFO - DAG(s) 'weatherbit_dag' retrieved from /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:43:16.633+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:43:16.632+0000] {dag.py:3211} INFO - Sync 1 DAGs
[2024-09-09T10:43:16.652+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:43:16.652+0000] {dag.py:4138} INFO - Setting next_dagrun for weatherbit_dag to 2024-09-09 00:00:00+00:00, run_after=2024-09-10 00:00:00+00:00
[2024-09-09T10:43:16.666+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/weatherbit_dag.py took 0.078 seconds
[2024-09-09T10:43:46.734+0000] {processor.py:186} INFO - Started process (PID=287) to work on /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:43:46.735+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/weatherbit_dag.py for tasks to queue
[2024-09-09T10:43:46.736+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:43:46.736+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:43:46.756+0000] {processor.py:925} INFO - DAG(s) 'weatherbit_dag' retrieved from /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:43:46.773+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:43:46.773+0000] {dag.py:3211} INFO - Sync 1 DAGs
[2024-09-09T10:43:46.790+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:43:46.790+0000] {dag.py:4138} INFO - Setting next_dagrun for weatherbit_dag to 2024-09-09 00:00:00+00:00, run_after=2024-09-10 00:00:00+00:00
[2024-09-09T10:43:46.802+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/weatherbit_dag.py took 0.072 seconds
[2024-09-09T10:44:16.946+0000] {processor.py:186} INFO - Started process (PID=295) to work on /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:44:16.946+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/weatherbit_dag.py for tasks to queue
[2024-09-09T10:44:16.948+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:44:16.948+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:44:16.973+0000] {processor.py:925} INFO - DAG(s) 'weatherbit_dag' retrieved from /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:44:16.991+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:44:16.991+0000] {dag.py:3211} INFO - Sync 1 DAGs
[2024-09-09T10:44:17.010+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:44:17.010+0000] {dag.py:4138} INFO - Setting next_dagrun for weatherbit_dag to 2024-09-09 00:00:00+00:00, run_after=2024-09-10 00:00:00+00:00
[2024-09-09T10:44:17.028+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/weatherbit_dag.py took 0.086 seconds
[2024-09-09T10:44:47.109+0000] {processor.py:186} INFO - Started process (PID=302) to work on /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:44:47.109+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/weatherbit_dag.py for tasks to queue
[2024-09-09T10:44:47.111+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:44:47.111+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:44:47.135+0000] {processor.py:925} INFO - DAG(s) 'weatherbit_dag' retrieved from /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:44:47.154+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:44:47.153+0000] {dag.py:3211} INFO - Sync 1 DAGs
[2024-09-09T10:44:47.171+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:44:47.171+0000] {dag.py:4138} INFO - Setting next_dagrun for weatherbit_dag to 2024-09-09 00:00:00+00:00, run_after=2024-09-10 00:00:00+00:00
[2024-09-09T10:44:47.183+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/weatherbit_dag.py took 0.078 seconds
[2024-09-09T10:45:17.311+0000] {processor.py:186} INFO - Started process (PID=309) to work on /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:45:17.311+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/weatherbit_dag.py for tasks to queue
[2024-09-09T10:45:17.313+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:45:17.313+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:45:17.335+0000] {processor.py:925} INFO - DAG(s) 'weatherbit_dag' retrieved from /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:45:17.352+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:45:17.352+0000] {dag.py:3211} INFO - Sync 1 DAGs
[2024-09-09T10:45:17.368+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:45:17.368+0000] {dag.py:4138} INFO - Setting next_dagrun for weatherbit_dag to 2024-09-09 00:00:00+00:00, run_after=2024-09-10 00:00:00+00:00
[2024-09-09T10:45:17.384+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/weatherbit_dag.py took 0.076 seconds
[2024-09-09T10:45:47.456+0000] {processor.py:186} INFO - Started process (PID=316) to work on /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:45:47.457+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/weatherbit_dag.py for tasks to queue
[2024-09-09T10:45:47.458+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:45:47.458+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:45:47.480+0000] {processor.py:925} INFO - DAG(s) 'weatherbit_dag' retrieved from /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:45:47.497+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:45:47.497+0000] {dag.py:3211} INFO - Sync 1 DAGs
[2024-09-09T10:45:47.514+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:45:47.514+0000] {dag.py:4138} INFO - Setting next_dagrun for weatherbit_dag to 2024-09-09 00:00:00+00:00, run_after=2024-09-10 00:00:00+00:00
[2024-09-09T10:45:47.527+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/weatherbit_dag.py took 0.075 seconds
[2024-09-09T10:46:17.648+0000] {processor.py:186} INFO - Started process (PID=323) to work on /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:46:17.649+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/weatherbit_dag.py for tasks to queue
[2024-09-09T10:46:17.651+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:46:17.650+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:46:17.672+0000] {processor.py:925} INFO - DAG(s) 'weatherbit_dag' retrieved from /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:46:17.688+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:46:17.688+0000] {dag.py:3211} INFO - Sync 1 DAGs
[2024-09-09T10:46:17.704+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:46:17.704+0000] {dag.py:4138} INFO - Setting next_dagrun for weatherbit_dag to 2024-09-09 00:00:00+00:00, run_after=2024-09-10 00:00:00+00:00
[2024-09-09T10:46:17.718+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/weatherbit_dag.py took 0.073 seconds
[2024-09-09T10:46:47.793+0000] {processor.py:186} INFO - Started process (PID=330) to work on /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:46:47.793+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/weatherbit_dag.py for tasks to queue
[2024-09-09T10:46:47.795+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:46:47.795+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:46:47.819+0000] {processor.py:925} INFO - DAG(s) 'weatherbit_dag' retrieved from /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:46:47.836+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:46:47.836+0000] {dag.py:3211} INFO - Sync 1 DAGs
[2024-09-09T10:46:47.855+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:46:47.855+0000] {dag.py:4138} INFO - Setting next_dagrun for weatherbit_dag to 2024-09-09 00:00:00+00:00, run_after=2024-09-10 00:00:00+00:00
[2024-09-09T10:46:47.870+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/weatherbit_dag.py took 0.081 seconds
[2024-09-09T10:47:17.976+0000] {processor.py:186} INFO - Started process (PID=337) to work on /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:47:17.977+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/weatherbit_dag.py for tasks to queue
[2024-09-09T10:47:17.978+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:47:17.978+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:47:18.000+0000] {processor.py:925} INFO - DAG(s) 'weatherbit_dag' retrieved from /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:47:18.016+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:47:18.016+0000] {dag.py:3211} INFO - Sync 1 DAGs
[2024-09-09T10:47:18.034+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:47:18.034+0000] {dag.py:4138} INFO - Setting next_dagrun for weatherbit_dag to 2024-09-09 00:00:00+00:00, run_after=2024-09-10 00:00:00+00:00
[2024-09-09T10:47:18.047+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/weatherbit_dag.py took 0.075 seconds
[2024-09-09T10:47:48.126+0000] {processor.py:186} INFO - Started process (PID=344) to work on /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:47:48.127+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/weatherbit_dag.py for tasks to queue
[2024-09-09T10:47:48.128+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:47:48.128+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:47:48.151+0000] {processor.py:925} INFO - DAG(s) 'weatherbit_dag' retrieved from /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:47:48.167+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:47:48.167+0000] {dag.py:3211} INFO - Sync 1 DAGs
[2024-09-09T10:47:48.185+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:47:48.185+0000] {dag.py:4138} INFO - Setting next_dagrun for weatherbit_dag to 2024-09-09 00:00:00+00:00, run_after=2024-09-10 00:00:00+00:00
[2024-09-09T10:47:48.199+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/weatherbit_dag.py took 0.076 seconds
[2024-09-09T10:48:18.337+0000] {processor.py:186} INFO - Started process (PID=351) to work on /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:48:18.338+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/weatherbit_dag.py for tasks to queue
[2024-09-09T10:48:18.339+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:48:18.339+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:48:18.361+0000] {processor.py:925} INFO - DAG(s) 'weatherbit_dag' retrieved from /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:48:18.378+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:48:18.378+0000] {dag.py:3211} INFO - Sync 1 DAGs
[2024-09-09T10:48:18.395+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:48:18.395+0000] {dag.py:4138} INFO - Setting next_dagrun for weatherbit_dag to 2024-09-09 00:00:00+00:00, run_after=2024-09-10 00:00:00+00:00
[2024-09-09T10:48:18.408+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/weatherbit_dag.py took 0.074 seconds
[2024-09-09T10:48:48.477+0000] {processor.py:186} INFO - Started process (PID=358) to work on /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:48:48.478+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/weatherbit_dag.py for tasks to queue
[2024-09-09T10:48:48.479+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:48:48.479+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:48:48.501+0000] {processor.py:925} INFO - DAG(s) 'weatherbit_dag' retrieved from /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:48:48.518+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:48:48.518+0000] {dag.py:3211} INFO - Sync 1 DAGs
[2024-09-09T10:48:48.534+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:48:48.534+0000] {dag.py:4138} INFO - Setting next_dagrun for weatherbit_dag to 2024-09-09 00:00:00+00:00, run_after=2024-09-10 00:00:00+00:00
[2024-09-09T10:48:48.546+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/weatherbit_dag.py took 0.073 seconds
[2024-09-09T10:49:18.668+0000] {processor.py:186} INFO - Started process (PID=365) to work on /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:49:18.668+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/weatherbit_dag.py for tasks to queue
[2024-09-09T10:49:18.670+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:49:18.670+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:49:18.695+0000] {processor.py:925} INFO - DAG(s) 'weatherbit_dag' retrieved from /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:49:18.711+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:49:18.711+0000] {dag.py:3211} INFO - Sync 1 DAGs
[2024-09-09T10:49:18.727+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:49:18.727+0000] {dag.py:4138} INFO - Setting next_dagrun for weatherbit_dag to 2024-09-09 00:00:00+00:00, run_after=2024-09-10 00:00:00+00:00
[2024-09-09T10:49:18.740+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/weatherbit_dag.py took 0.075 seconds
[2024-09-09T10:49:48.809+0000] {processor.py:186} INFO - Started process (PID=373) to work on /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:49:48.810+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/weatherbit_dag.py for tasks to queue
[2024-09-09T10:49:48.812+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:49:48.812+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:49:48.832+0000] {processor.py:925} INFO - DAG(s) 'weatherbit_dag' retrieved from /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:49:48.848+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:49:48.848+0000] {dag.py:3211} INFO - Sync 1 DAGs
[2024-09-09T10:49:48.865+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:49:48.865+0000] {dag.py:4138} INFO - Setting next_dagrun for weatherbit_dag to 2024-09-09 00:00:00+00:00, run_after=2024-09-10 00:00:00+00:00
[2024-09-09T10:49:48.877+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/weatherbit_dag.py took 0.072 seconds
[2024-09-09T10:50:18.997+0000] {processor.py:186} INFO - Started process (PID=381) to work on /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:50:18.998+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/weatherbit_dag.py for tasks to queue
[2024-09-09T10:50:19.000+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:50:18.999+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:50:19.021+0000] {processor.py:925} INFO - DAG(s) 'weatherbit_dag' retrieved from /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:50:19.037+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:50:19.037+0000] {dag.py:3211} INFO - Sync 1 DAGs
[2024-09-09T10:50:19.055+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:50:19.054+0000] {dag.py:4138} INFO - Setting next_dagrun for weatherbit_dag to 2024-09-09 00:00:00+00:00, run_after=2024-09-10 00:00:00+00:00
[2024-09-09T10:50:19.066+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/weatherbit_dag.py took 0.072 seconds
[2024-09-09T10:50:49.140+0000] {processor.py:186} INFO - Started process (PID=388) to work on /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:50:49.141+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/weatherbit_dag.py for tasks to queue
[2024-09-09T10:50:49.143+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:50:49.143+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:50:49.166+0000] {processor.py:925} INFO - DAG(s) 'weatherbit_dag' retrieved from /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:50:49.183+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:50:49.183+0000] {dag.py:3211} INFO - Sync 1 DAGs
[2024-09-09T10:50:49.200+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:50:49.199+0000] {dag.py:4138} INFO - Setting next_dagrun for weatherbit_dag to 2024-09-09 00:00:00+00:00, run_after=2024-09-10 00:00:00+00:00
[2024-09-09T10:50:49.211+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/weatherbit_dag.py took 0.075 seconds
[2024-09-09T10:51:19.324+0000] {processor.py:186} INFO - Started process (PID=395) to work on /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:51:19.325+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/weatherbit_dag.py for tasks to queue
[2024-09-09T10:51:19.326+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:51:19.326+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:51:19.350+0000] {processor.py:925} INFO - DAG(s) 'weatherbit_dag' retrieved from /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:51:19.367+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:51:19.367+0000] {dag.py:3211} INFO - Sync 1 DAGs
[2024-09-09T10:51:19.383+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:51:19.383+0000] {dag.py:4138} INFO - Setting next_dagrun for weatherbit_dag to 2024-09-09 00:00:00+00:00, run_after=2024-09-10 00:00:00+00:00
[2024-09-09T10:51:19.395+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/weatherbit_dag.py took 0.075 seconds
[2024-09-09T10:51:49.461+0000] {processor.py:186} INFO - Started process (PID=402) to work on /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:51:49.462+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/weatherbit_dag.py for tasks to queue
[2024-09-09T10:51:49.463+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:51:49.463+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:51:49.486+0000] {processor.py:925} INFO - DAG(s) 'weatherbit_dag' retrieved from /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:51:49.503+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:51:49.502+0000] {dag.py:3211} INFO - Sync 1 DAGs
[2024-09-09T10:51:49.519+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:51:49.519+0000] {dag.py:4138} INFO - Setting next_dagrun for weatherbit_dag to 2024-09-09 00:00:00+00:00, run_after=2024-09-10 00:00:00+00:00
[2024-09-09T10:51:49.531+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/weatherbit_dag.py took 0.074 seconds
[2024-09-09T10:52:19.654+0000] {processor.py:186} INFO - Started process (PID=409) to work on /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:52:19.655+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/weatherbit_dag.py for tasks to queue
[2024-09-09T10:52:19.657+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:52:19.656+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:52:19.680+0000] {processor.py:925} INFO - DAG(s) 'weatherbit_dag' retrieved from /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:52:19.697+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:52:19.696+0000] {dag.py:3211} INFO - Sync 1 DAGs
[2024-09-09T10:52:19.713+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:52:19.713+0000] {dag.py:4138} INFO - Setting next_dagrun for weatherbit_dag to 2024-09-09 00:00:00+00:00, run_after=2024-09-10 00:00:00+00:00
[2024-09-09T10:52:19.727+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/weatherbit_dag.py took 0.076 seconds
[2024-09-09T10:52:49.796+0000] {processor.py:186} INFO - Started process (PID=416) to work on /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:52:49.797+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/weatherbit_dag.py for tasks to queue
[2024-09-09T10:52:49.798+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:52:49.798+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:52:49.821+0000] {processor.py:925} INFO - DAG(s) 'weatherbit_dag' retrieved from /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:52:49.838+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:52:49.838+0000] {dag.py:3211} INFO - Sync 1 DAGs
[2024-09-09T10:52:49.856+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:52:49.855+0000] {dag.py:4138} INFO - Setting next_dagrun for weatherbit_dag to 2024-09-09 00:00:00+00:00, run_after=2024-09-10 00:00:00+00:00
[2024-09-09T10:52:49.868+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/weatherbit_dag.py took 0.076 seconds
[2024-09-09T10:53:20.018+0000] {processor.py:186} INFO - Started process (PID=423) to work on /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:53:20.019+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/weatherbit_dag.py for tasks to queue
[2024-09-09T10:53:20.020+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:53:20.020+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:53:20.043+0000] {processor.py:925} INFO - DAG(s) 'weatherbit_dag' retrieved from /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:53:20.059+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:53:20.059+0000] {dag.py:3211} INFO - Sync 1 DAGs
[2024-09-09T10:53:20.076+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:53:20.075+0000] {dag.py:4138} INFO - Setting next_dagrun for weatherbit_dag to 2024-09-09 00:00:00+00:00, run_after=2024-09-10 00:00:00+00:00
[2024-09-09T10:53:20.089+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/weatherbit_dag.py took 0.074 seconds
[2024-09-09T10:53:50.159+0000] {processor.py:186} INFO - Started process (PID=430) to work on /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:53:50.159+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/weatherbit_dag.py for tasks to queue
[2024-09-09T10:53:50.161+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:53:50.161+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:53:50.184+0000] {processor.py:925} INFO - DAG(s) 'weatherbit_dag' retrieved from /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:53:50.201+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:53:50.200+0000] {dag.py:3211} INFO - Sync 1 DAGs
[2024-09-09T10:53:50.217+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:53:50.217+0000] {dag.py:4138} INFO - Setting next_dagrun for weatherbit_dag to 2024-09-09 00:00:00+00:00, run_after=2024-09-10 00:00:00+00:00
[2024-09-09T10:53:50.229+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/weatherbit_dag.py took 0.075 seconds
[2024-09-09T10:54:20.352+0000] {processor.py:186} INFO - Started process (PID=437) to work on /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:54:20.352+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/weatherbit_dag.py for tasks to queue
[2024-09-09T10:54:20.354+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:54:20.354+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:54:20.376+0000] {processor.py:925} INFO - DAG(s) 'weatherbit_dag' retrieved from /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:54:20.393+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:54:20.393+0000] {dag.py:3211} INFO - Sync 1 DAGs
[2024-09-09T10:54:20.409+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:54:20.409+0000] {dag.py:4138} INFO - Setting next_dagrun for weatherbit_dag to 2024-09-09 00:00:00+00:00, run_after=2024-09-10 00:00:00+00:00
[2024-09-09T10:54:20.423+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/weatherbit_dag.py took 0.075 seconds
[2024-09-09T10:54:50.495+0000] {processor.py:186} INFO - Started process (PID=444) to work on /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:54:50.496+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/weatherbit_dag.py for tasks to queue
[2024-09-09T10:54:50.498+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:54:50.498+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:54:50.521+0000] {processor.py:925} INFO - DAG(s) 'weatherbit_dag' retrieved from /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:54:50.538+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:54:50.538+0000] {dag.py:3211} INFO - Sync 1 DAGs
[2024-09-09T10:54:50.556+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:54:50.555+0000] {dag.py:4138} INFO - Setting next_dagrun for weatherbit_dag to 2024-09-09 00:00:00+00:00, run_after=2024-09-10 00:00:00+00:00
[2024-09-09T10:54:50.568+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/weatherbit_dag.py took 0.077 seconds
[2024-09-09T10:55:20.688+0000] {processor.py:186} INFO - Started process (PID=451) to work on /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:55:20.688+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/weatherbit_dag.py for tasks to queue
[2024-09-09T10:55:20.690+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:55:20.690+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:55:20.715+0000] {processor.py:925} INFO - DAG(s) 'weatherbit_dag' retrieved from /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:55:20.733+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:55:20.733+0000] {dag.py:3211} INFO - Sync 1 DAGs
[2024-09-09T10:55:20.753+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:55:20.753+0000] {dag.py:4138} INFO - Setting next_dagrun for weatherbit_dag to 2024-09-09 00:00:00+00:00, run_after=2024-09-10 00:00:00+00:00
[2024-09-09T10:55:20.766+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/weatherbit_dag.py took 0.081 seconds
[2024-09-09T10:55:50.835+0000] {processor.py:186} INFO - Started process (PID=458) to work on /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:55:50.835+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/weatherbit_dag.py for tasks to queue
[2024-09-09T10:55:50.837+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:55:50.836+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:55:50.859+0000] {processor.py:925} INFO - DAG(s) 'weatherbit_dag' retrieved from /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:55:50.875+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:55:50.875+0000] {dag.py:3211} INFO - Sync 1 DAGs
[2024-09-09T10:55:50.893+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:55:50.893+0000] {dag.py:4138} INFO - Setting next_dagrun for weatherbit_dag to 2024-09-09 00:00:00+00:00, run_after=2024-09-10 00:00:00+00:00
[2024-09-09T10:55:50.906+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/weatherbit_dag.py took 0.075 seconds
[2024-09-09T10:56:21.033+0000] {processor.py:186} INFO - Started process (PID=465) to work on /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:56:21.033+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/weatherbit_dag.py for tasks to queue
[2024-09-09T10:56:21.035+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:56:21.035+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:56:21.059+0000] {processor.py:925} INFO - DAG(s) 'weatherbit_dag' retrieved from /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:56:21.076+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:56:21.076+0000] {dag.py:3211} INFO - Sync 1 DAGs
[2024-09-09T10:56:21.092+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:56:21.092+0000] {dag.py:4138} INFO - Setting next_dagrun for weatherbit_dag to 2024-09-09 00:00:00+00:00, run_after=2024-09-10 00:00:00+00:00
[2024-09-09T10:56:21.106+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/weatherbit_dag.py took 0.076 seconds
[2024-09-09T10:56:51.179+0000] {processor.py:186} INFO - Started process (PID=472) to work on /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:56:51.180+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/weatherbit_dag.py for tasks to queue
[2024-09-09T10:56:51.181+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:56:51.181+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:56:51.205+0000] {processor.py:925} INFO - DAG(s) 'weatherbit_dag' retrieved from /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:56:51.224+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:56:51.224+0000] {dag.py:3211} INFO - Sync 1 DAGs
[2024-09-09T10:56:51.241+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:56:51.241+0000] {dag.py:4138} INFO - Setting next_dagrun for weatherbit_dag to 2024-09-09 00:00:00+00:00, run_after=2024-09-10 00:00:00+00:00
[2024-09-09T10:56:51.255+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/weatherbit_dag.py took 0.080 seconds
[2024-09-09T10:57:21.380+0000] {processor.py:186} INFO - Started process (PID=479) to work on /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:57:21.381+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/weatherbit_dag.py for tasks to queue
[2024-09-09T10:57:21.382+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:57:21.382+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:57:21.406+0000] {processor.py:925} INFO - DAG(s) 'weatherbit_dag' retrieved from /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:57:21.423+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:57:21.423+0000] {dag.py:3211} INFO - Sync 1 DAGs
[2024-09-09T10:57:21.441+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:57:21.440+0000] {dag.py:4138} INFO - Setting next_dagrun for weatherbit_dag to 2024-09-09 00:00:00+00:00, run_after=2024-09-10 00:00:00+00:00
[2024-09-09T10:57:21.454+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/weatherbit_dag.py took 0.077 seconds
[2024-09-09T10:57:51.530+0000] {processor.py:186} INFO - Started process (PID=486) to work on /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:57:51.530+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/weatherbit_dag.py for tasks to queue
[2024-09-09T10:57:51.532+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:57:51.532+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:57:51.558+0000] {processor.py:925} INFO - DAG(s) 'weatherbit_dag' retrieved from /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:57:51.576+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:57:51.576+0000] {dag.py:3211} INFO - Sync 1 DAGs
[2024-09-09T10:57:51.595+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:57:51.594+0000] {dag.py:4138} INFO - Setting next_dagrun for weatherbit_dag to 2024-09-09 00:00:00+00:00, run_after=2024-09-10 00:00:00+00:00
[2024-09-09T10:57:51.609+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/weatherbit_dag.py took 0.084 seconds
[2024-09-09T10:58:21.775+0000] {processor.py:186} INFO - Started process (PID=493) to work on /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:58:21.775+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/weatherbit_dag.py for tasks to queue
[2024-09-09T10:58:21.777+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:58:21.777+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:58:21.800+0000] {processor.py:925} INFO - DAG(s) 'weatherbit_dag' retrieved from /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:58:21.818+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:58:21.817+0000] {dag.py:3211} INFO - Sync 1 DAGs
[2024-09-09T10:58:21.836+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:58:21.836+0000] {dag.py:4138} INFO - Setting next_dagrun for weatherbit_dag to 2024-09-09 00:00:00+00:00, run_after=2024-09-10 00:00:00+00:00
[2024-09-09T10:58:21.849+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/weatherbit_dag.py took 0.079 seconds
[2024-09-09T10:58:51.925+0000] {processor.py:186} INFO - Started process (PID=500) to work on /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:58:51.925+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/weatherbit_dag.py for tasks to queue
[2024-09-09T10:58:51.927+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:58:51.927+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:58:51.952+0000] {processor.py:925} INFO - DAG(s) 'weatherbit_dag' retrieved from /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:58:51.969+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:58:51.969+0000] {dag.py:3211} INFO - Sync 1 DAGs
[2024-09-09T10:58:51.990+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:58:51.989+0000] {dag.py:4138} INFO - Setting next_dagrun for weatherbit_dag to 2024-09-09 00:00:00+00:00, run_after=2024-09-10 00:00:00+00:00
[2024-09-09T10:58:52.006+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/weatherbit_dag.py took 0.085 seconds
[2024-09-09T10:59:22.135+0000] {processor.py:186} INFO - Started process (PID=507) to work on /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:59:22.135+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/weatherbit_dag.py for tasks to queue
[2024-09-09T10:59:22.137+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:59:22.137+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:59:22.159+0000] {processor.py:925} INFO - DAG(s) 'weatherbit_dag' retrieved from /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:59:22.175+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:59:22.175+0000] {dag.py:3211} INFO - Sync 1 DAGs
[2024-09-09T10:59:22.193+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:59:22.192+0000] {dag.py:4138} INFO - Setting next_dagrun for weatherbit_dag to 2024-09-09 00:00:00+00:00, run_after=2024-09-10 00:00:00+00:00
[2024-09-09T10:59:22.207+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/weatherbit_dag.py took 0.077 seconds
[2024-09-09T10:59:52.280+0000] {processor.py:186} INFO - Started process (PID=514) to work on /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:59:52.281+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/weatherbit_dag.py for tasks to queue
[2024-09-09T10:59:52.283+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:59:52.282+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:59:52.295+0000] {logging_mixin.py:190} INFO - [2024-09-09T10:59:52.293+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/weatherbit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/weatherbit_dag.py", line 5, in <module>
    from operators.weatherbit_operator import WeatherbitOperator
ModuleNotFoundError: No module named 'operators'
[2024-09-09T10:59:52.296+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T10:59:52.310+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/weatherbit_dag.py took 0.034 seconds
[2024-09-09T13:11:55.014+0000] {processor.py:186} INFO - Started process (PID=54) to work on /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T13:11:55.015+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/weatherbit_dag.py for tasks to queue
[2024-09-09T13:11:55.016+0000] {logging_mixin.py:190} INFO - [2024-09-09T13:11:55.016+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T13:11:56.106+0000] {logging_mixin.py:190} INFO - [2024-09-09T13:11:56.103+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/weatherbit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/weatherbit_dag.py", line 6, in <module>
    from hooks.our_ingestion_api_hook import IngestionApiHook
ModuleNotFoundError: No module named 'hooks.our_ingestion_api_hook'
[2024-09-09T13:11:56.107+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T13:11:56.120+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/weatherbit_dag.py took 1.109 seconds
[2024-09-09T13:12:26.186+0000] {processor.py:186} INFO - Started process (PID=62) to work on /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T13:12:26.187+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/weatherbit_dag.py for tasks to queue
[2024-09-09T13:12:26.188+0000] {logging_mixin.py:190} INFO - [2024-09-09T13:12:26.188+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T13:12:26.554+0000] {logging_mixin.py:190} INFO - [2024-09-09T13:12:26.552+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/weatherbit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/weatherbit_dag.py", line 6, in <module>
    from hooks.our_ingestion_api_hook import IngestionApiHook
ModuleNotFoundError: No module named 'hooks.our_ingestion_api_hook'
[2024-09-09T13:12:26.556+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T13:12:26.567+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/weatherbit_dag.py took 0.385 seconds
[2024-09-09T13:12:56.726+0000] {processor.py:186} INFO - Started process (PID=70) to work on /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T13:12:56.726+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/weatherbit_dag.py for tasks to queue
[2024-09-09T13:12:56.728+0000] {logging_mixin.py:190} INFO - [2024-09-09T13:12:56.727+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T13:12:57.086+0000] {logging_mixin.py:190} INFO - [2024-09-09T13:12:57.083+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/weatherbit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/weatherbit_dag.py", line 6, in <module>
    from hooks.our_ingestion_api_hook import IngestionApiHook
ModuleNotFoundError: No module named 'hooks.our_ingestion_api_hook'
[2024-09-09T13:12:57.087+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T13:12:57.101+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/weatherbit_dag.py took 0.379 seconds
[2024-09-09T13:13:27.259+0000] {processor.py:186} INFO - Started process (PID=78) to work on /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T13:13:27.260+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/weatherbit_dag.py for tasks to queue
[2024-09-09T13:13:27.261+0000] {logging_mixin.py:190} INFO - [2024-09-09T13:13:27.261+0000] {dagbag.py:587} INFO - Filling up the DagBag from /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T13:13:27.590+0000] {logging_mixin.py:190} INFO - [2024-09-09T13:13:27.587+0000] {dagbag.py:386} ERROR - Failed to import: /opt/airflow/dags/weatherbit_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 382, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/weatherbit_dag.py", line 6, in <module>
    from hooks.our_ingestion_api_hook import IngestionApiHook
ModuleNotFoundError: No module named 'hooks.our_ingestion_api_hook'
[2024-09-09T13:13:27.590+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/weatherbit_dag.py
[2024-09-09T13:13:27.602+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/weatherbit_dag.py took 0.346 seconds
